ramls/schemas/dto/filteringConditionValue.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Filtering condition DTO Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "description": "Filtering condition name",
+      "type": "string"
+    },
+    "values": {
+      "description": "Filtering condition value",
+      "type": "array"
+    }
+  }
```

Comment:
I'm guessing both of these are required fields?

Reply:
No, this never will be taken this from UI or any else via post request for example, this model is used for response only and i use model instead of manual json composing cause vertx JsonObject honestly isn't good into composing json structure correctly, I can show you the particular example if you wish) So, there are no necessity to validate its fields for values presence.

Reply:
OK, that's fine.  I'm not saying you have to, but consider adding it though as I don't think it hurts anything and it might be nice to be able to look at the schema and know that both properties will always be present.

Reply:
> OK, that's fine. I'm not saying you have to, but consider adding it though as I don't think it hurts anything and it might be nice to be able to look at the schema and know that both properties will always be present.
Okay, i will add this before merge, thank you